scripts/dockerfiles/readiness-probe.sh

```diff
@@ -16,12 +16,14 @@
 
 set -o errexit -o nounset
 
+# Grab monit port
+monit_port=$(cat /etc/monitrc | grep "httpd port" | awk '{print $4}')
```

Comment:
That's the same as `awk '/httpd port/ { print $4 }' /etc/monitrc`?

Reply:
I believe so, yes. Guilty of similar constructions myself. Why use one process if you can have three ;)

Reply:
@mook-as @viovanov asked me to handle this since he's on the road.

I verified that your suggestion works and changed the code accordingly.